llpc/test/shaderdb/core/OverrideThreadGroupSize16X16X1.spvasm

```diff
@@ -0,0 +1,102 @@
+; BEGIN_SWIZZLETEST
+; RUN: amdllpc -v %gfxip %s --override-threadGroupSizeX=16 --override-threadGroupSizeY=16 --override-threadGroupSizeZ=1 | FileCheck -check-prefix=OVERRIDETHREADGROUPTEST %s
+; OVERRIDETHREADGROUPTEST-LABEL: {{^// LLPC}} pipeline before-patching results
+; OVERRIDETHREADGROUPTEST: !0 = !{i32 16, i32 16, i32 1}
```

Comment:
What is this metadata associated with? I think the test should also match the related function signature and perhaps other metadata nodes, so that we can be confident that this node corresponds to thread group sizes.

Reply:
OK. Thanks for the suggestion. I would like adding "!llpc.compute.mode = !{!0}" here, which will check whether the !0 is the metadata node for compute mode.